map/map_tests/CMakeLists.txt

```diff
@@ -19,7 +19,12 @@ set(SRC
   working_time_tests.cpp
 )
 
-omim_add_test_with_qt_event_loop(${PROJECT_NAME} ${SRC})
+omim_add_test(
+  NAME ${PROJECT_NAME}
+  SRC ${SRC}
+  REQUIRE_QT
```

Comment:
Can these vars be passed to omim_add_test without modifying it by adding NAME and SRC?